src/TGameDetails.h

```diff
@@ -62,7 +62,15 @@ class TGameDetails
     }
 
     // clang-format off
-    // games are to be added here in alphabetical order
+    //
+    // Games are to be added here in alphabetical order and this format:
+    // - Name
+    // - Address to connect to
+    // - Port to connect on
+    // - Secure connection possible?
+    // - Website or other URLs
+    // - Path to the profile icon
+    // - Text to use for description
```

Comment:
Not needed - I have already documented the first entry - if you insist that it needs to be done here, like this, then please remove corresponding comments from that first entry.
```suggestion
    // games are to be added here in alphabetical order
```

Reply:
No hard feelings on this one, but I agree it maybe should not be documented twice. 
Probably a matter of taste, but I found the inline-documentation less readable albeit more informative.
Leaving this open for a moment for others to review and maybe add their opinion as well.
Then again, this probably should not be part of adding a game at all, but a PR of its own.